projects/igniteui-angular/src/lib/grids/common/events.ts

```diff
@@ -29,6 +29,7 @@ export interface IGridEditEventArgs extends CancelableEventArgs, IBaseEventArgs
     oldValue: any;
     newValue?: any;
     event?: Event;
+    columnField?: string;
```

Comment:
IMO it maybe better to actually return the column instance when emitting with cell editing?
@damyanpetev your thoughts about this?